src/components/PokeCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { mockPokemonData } from "../mock/pokeData";
```

Comment:
I would be better to load this mock data out of the component and then pass them as properties.  In this way, you could unit test different scenarios and use this component in other places without repeating your self.